onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -94,13 +91,13 @@ class NodeRepo {
     return node_repo;
   }
 
-  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr node_ptr) {
+  onnxruntime::Status AddNode(const onnxruntime::OpKernel* kernel, NodePtr&& node_ptr, ArgPtrs&& args) {
     std::lock_guard<std::mutex> guard(mutex_);
-    auto iter = node_map_.find(kernel);
-    if (iter != node_map_.end()) {
+    auto iter = resource_map_.find(kernel);
```

Comment:
unordered_map has some other useful functions, e.g., try_emplace, that can express "insert if not there" concisely:

```c++
  const auto inserted = resource_map_.try_emplace(kernel, std::move(node_ptr), std::move(args)).second;
  ORT_RETURN_IF_NOT(inserted, "kernel already mapped to existing node");
```

Reply:
Done.